nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -103,31 +103,18 @@ public static void ModifyAvatarActionPoint(Address avatarAddress, int actionPoin
 
         #region Avatar / AddItem
 
-        public static void AddItem(Address avatarAddress, Guid guid, bool resetState = true)
-        {
-            var modifier = new AvatarInventoryNonFungibleItemRemover(guid);
-            LocalLayer.Instance.Remove(avatarAddress, modifier);
-
-            if (!resetState)
-            {
-                return;
-            }
-
-            TryResetLoadedAvatarState(avatarAddress, out _, out _);
-        }
-
         public static void AddItem(
             Address avatarAddress,
-            HashDigest<SHA256> id,
-            int count,
-            bool resetState = true)
+            Guid tradableId,
+            int count = 1,
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.